akka-docs/src/main/paradox/stream/operators/Source/actorRefWithAck.md

```diff
@@ -0,0 +1,33 @@
+# actorRefWithAck
+
+Materialize an `ActorRef`; sending messages to it will emit them on the stream. The source acknowledges reception after emitting a message, to provide back pressure from the source.
+
+@ref[Source operators](../index.md#source-operators)
+
+@@@ div { .group-scala }
+## Signature
+
+@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #actorRefWithAck }
+@@@
+
+## Description
+
+Materialize an `ActorRef`, sending messages to it will emit them on the stream. The actor responds with the provided ack message
+once the element could be emitted alowing for backpressure from the source.
```

Comment:
clarify that a new message can't be sent until the previous has been acknowledged